### ISSUE TITLE
Staging Review finding: Add list semantics for service list items

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#106800]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** AEDP
**Product:** Design Patterns
**Feature:** Critical Information and Service List Item Components

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

As currently coded, service list items aren't actually HTML list items.

When screen reader users move through a list of items, HTML list semantics (eg. an unordered list `<ul>` with each item given as a `<li>`) provide screen reader users with:
- Information about the number of items in the list (eg. "list, six items")
- The ability to move up and down through the list, item by item
- Clearly-announced boundaries for the start and end of a list item

The heading structure in the service list item component provides a lot of navigation support for screen reader users, but this is a "yes and" situation where headings plus list semantics can give users extra options for understadning and and navigating content.

## Recommended action

Introduce list semantics, with each service list item wrapped in a `li` within a `ul`. This may require some re-thinking of how the component is organized, since there's currently no containing component. But I'd recommend looking at the [va-accordion component](https://design.va.gov/storybook/?path=/docs/uswds-va-accordion--docs) for inspiration. Accordions aren't coded as lists, but they are structured as a containing component with item components inside of the container, which I think would be a good approach here.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by msbtterswrth** [comment]
@bellepx0 will confirm that this does infact use ul/li markup.

**Comment by msbtterswrth** [comment]
We've updated this component to use ul/li wrappers around the custom element in this [PR](https://github.com/department-of-veterans-affairs/component-library/pull/1563).
